packages/yourloops/components/alarm/alarms-content-configuration.hook.tsx

```diff
@@ -0,0 +1,234 @@
+    })
+  }
+
+  const resetToTeamDefaultValues = (): void => {
+    if (!patient) {
+      throw Error('This action cannot be done if the patient is undefined')
+    }
+    const monitoredTeam = PatientUtils.getRemoteMonitoringTeam(patient)
+    const team = teamHook.getTeam(monitoredTeam.teamId)
+    if (!team) {
+      throw Error(`Cannot find team with id ${monitoredTeam.teamId}`)
+    }
+    const defaultMonitoring = team.monitoring
+    if (!defaultMonitoring?.parameters) {
+      throw Error('The given team has no monitoring values')
+    }
+    setHighBg({ ...highBg, value: defaultMonitoring.parameters.highBg })
+    setVeryLowBg({ ...veryLowBg, value: defaultMonitoring.parameters.veryLowBg })
+    setLowBg({ ...lowBg, value: defaultMonitoring.parameters.lowBg })
+    setNonDataTxThreshold({ ...nonDataTxThreshold, value: defaultMonitoring.parameters.nonDataTxThreshold })
+    setOutOfRangeThreshold({ ...outOfRangeThreshold, value: defaultMonitoring.parameters.outOfRangeThreshold })
+    setHypoThreshold({ ...hypoThreshold, value: defaultMonitoring.parameters.hypoThreshold })
+  }
+
+  const save = (): void => {
+    if (
+      lowBg.value !== undefined &&
+      highBg.value !== undefined &&
+      veryLowBg.value !== undefined &&
+      outOfRangeThreshold.value !== undefined &&
+      nonDataTxThreshold.value !== undefined &&
+      hypoThreshold.value !== undefined
+    ) {
+      const reportingPeriod = (monitoring?.parameters?.reportingPeriod && monitoring?.parameters?.reportingPeriod > 0) ? monitoring?.parameters?.reportingPeriod : 55
+      const monitoringUpdated: Monitoring = {
+        enabled: monitoring?.enabled ?? true,
+        status: monitoring?.status,
+        monitoringEnd: monitoring?.monitoringEnd,
+        parameters: {
+          bgUnit: monitoring?.parameters?.bgUnit ?? UNITS_TYPE.MGDL,
+          lowBg: lowBg.value,
+          highBg: highBg.value,
+          outOfRangeThreshold: outOfRangeThreshold.value,
+          veryLowBg: veryLowBg.value,
+          hypoThreshold: hypoThreshold.value,
+          nonDataTxThreshold: nonDataTxThreshold.value,
+          reportingPeriod
+        }
+      }
+      onSave(monitoringUpdated)
+    } else {
+      throw Error('Cannot update team monitoring as some values are not defined')
+    }
+  }
+
+  const onBasicDropdownSelect = (value: string, setValue: React.Dispatch<{ value?: number, error: boolean }>): void => {
+    const valueAsNumber = +value.slice(0, -1)
+    setValue({
+      value: valueAsNumber,
+      error: false
+    })
+  }
+
+  return {
+    isError,
+    lowBg,
+    saveButtonDisabled,
+    onBasicDropdownSelect,
+    save,
+    resetToTeamDefaultValues,
+    onChange,
+    veryLowBg,
+    highBg,
+    hypoThreshold,
+    nonDataTxThreshold,
+    outOfRangeThreshold,
+    setLowBg,
+    setHighBg,
+    setOutOfRangeThreshold,
+    setHypoThreshold,
+    setVeryLowBg,
+    setNonDataTxThreshold,
+    MIN_HIGH_BG,
```

Comment:
MIN_HIGH_BG,
    MAX_HIGH_BG,
    MAX_LOW_BG,
    MAX_VERY_LOW_BG,
    MIN_VERY_LOW_BG,
    MIN_LOW_BG,
    PERCENTAGES,

All of these do not have to be returned by this custom hook. Those are exported variables and can be directly imported wherever we want them. They do not depend on the state of the hook